tests/unit/models/students-test.js

```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
+
+moduleForModel('students', 'Unit | Model | students', {
+  // Specify the other units that are required for this test.
```

Comment:
these were auto generated by the ember generator,  eventually planning on having tests, so for the time being, the comments remain.